qcut/docs/issues/critical-analysis/fix-hybrid-architecture-migration-plan.md

```
@@ -0,0 +1,1209 @@
+- **State**: Zustand stores
+- ~~**API**: Next.js API routes~~ (Migrated to Electron IPC)
+
+## Development
+```bash
+# Start development server
+bun run dev
+
+# Start Electron development
+bun run electron:dev
+```
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)